apps/obs-web/src/views/ErrorManagement.vue

```diff
@@ -68,9 +70,27 @@
                     ></q-btn-toggle>
                 <InputText type="text" v-else v-model="filters[col.field]" class="p-column-filter" placeholder="" debounce="2000"/>
             </template>
-            <template #body="slotProps">
+            <template v-if="col.isEditable" #editor="slotProps">
                 <div>
-                    {{ getVal(slotProps.data, col.field, col.header) }}
+                    <InputText
+                        type="text"
+                        v-model="cellVal"
+                        @change="onCellEdit($event.target.value, slotProps, col.type)"
+                    ></InputText>
+                </div>
+            </template>
+            <template #body="slotProps" v-else>
+                <div v-if="toggleFields.includes(col.field)" style="text-align: center">
+                    <q-checkbox v-model="slotProps.data[col.field]" dense
```

Comment:
If the error doesn't exist for a program should we display N/A instead of an empty checkbox? With the checkbox it seems like there's a way to turn on, but if the error doesn't exist in the program, or is irrelevant maybe we should show something else.

Reply:
We should discuss.  I'm not sure.